fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/ui/struts/action/research/researchUnit/UnitFileBean.java

```diff
@@ -111,4 +118,15 @@ public void setTags(String tags) {
         this.tags = tags;
     }
 
+    public List<Group> getGroups() {
+        String[] groupsExpressions = getGroup().getExpression().split("[ ][|][ ]");
```

Comment:
This is not an acceptable solution for this.
If the semantics is always an OR you can keep the list of groups in a field, and the getGroup method reduces it with or.